pyaedt/modeler/circuits/object3dcircuit.py

```diff
@@ -871,6 +872,37 @@ def change_property(self, property_name, names=None):
             return self._oeditor.ChangeProperty(vOut)
         return False
 
+    @pyaedt_function_handler()
+    def enforce_touchstone_model_passive(self):
+        """Enforce touchstone model passive.
+
+        Returns
+        -------
+        bool
```

Comment:
You should add something like (check other methods):
 bool
            ``True`` when successful, ``False`` when failed.

Reply:
@HaiwenZhang Please in the docstring add this:

        Returns
        -------
        bool
            ``True`` when successful, ``False`` when failed.